packages/react/src/providers/helio/HelioContext.tsx

```diff
@@ -69,14 +97,105 @@ export const useHelioProvider = () => {
     setPaymentDetails(result || {});
   };
 
-  useEffect(() => {
-    setIsCustomerDetailsRequired(checkCustomerDetailsRequired());
-  }, [paymentDetails]);
-
   const initCluster = (initialCluster: Cluster) => {
     setCluster(initialCluster);
   };
 
+  const getTokenSwapCurrencies = async () => {
+    setTokenSwapLoading(true);
+
+    const mintAddress: string | undefined =
+      paymentDetails?.currency?.mintAddress;
+
+    const SOL_MINT = '11111111111111111111111111111111';
```

Comment:
you can get this from our Public API, plus the currency service can be moved here

Reply:
we have a constant in common package, but getting error with using constants from there. if it's an easy fix let's discuss and fix. if not let's do during refactoring